src/atomate2/abinit/schemas/calculation.py

```diff
@@ -237,13 +280,23 @@ def from_abinit_files(
         abinit_gsr_file = dir_name / abinit_gsr_file
         abinit_log_file = dir_name / abinit_log_file
         abinit_abort_file = dir_name / abinit_abort_file
-
-        abinit_gsr = GsrFile.from_file(abinit_gsr_file)
+        abinit_out_file = dir_name / abinit_out_file
+
+        output_doc = None
+        if abinit_out_file.exists():
+            abinit_out = AbinitOutputFile.from_file(abinit_out_file)
+            output_doc = CalculationOutput.from_abinit_out(abinit_out)
+        if abinit_gsr_file.exists():
```

Comment:
Would it be better to reorganize the check with
```python
if abinit_gsr_file.exists():
    ...
elif abinit_out_file.exists():
    ...
else:
    raise ...
```
Or are there cases where it makes sense that `output_doc` is `None`?

Reply:
Then wouldn't it be better to do like I mentioned above? Otherwise if for whatever reason the files are not present you risk to get the output created but empty. Which will be difficult to debug.

Reply:
but ```output_doc``` needs to be None if there is not output file of any kind no?

Reply:
Then I am confused. 😄 In the reply above you said:
>But `output_doc` should never be None I think.

My point is that if the code gets to that point for some reason, but the files are not present, what should happen? With the current version of the code it seems that the output will just be empty, but no error is raised. I would expect that there should always be some kind of output. In that case it could be better to explicitly raise an error. But if there is a use case where no output file is present and it still makes sense then it can also be this way.

Reply:
Okay I got confused as well :')
My point was that ```output_doc``` should not be empty because ```run.abo``` will always be there (I think) which is not the case for the GSR output. Your suggested implementation is fine if ```output_doc``` is iteratively updated with the different output files but it should not depend only on one output file, that was my point. And IF there is no ```run.abo``` I agree that raising an error makes sense if there is no case that does not output a ```run.abo```.